### ISSUE TITLE
IS UI

### ISSUE THREAD

**Issue by HarshKhandeparkar** [HarshKhandeparkar/image-sequencer-ui#2]
**Main Issue**: publiclab/image-sequencer#1238

Current Status
- Scaffolding ready

**Comment by HarshKhandeparkar** [comment]
@harshithpabbati

**Comment by HarshKhandeparkar** [comment]
If anyone else wants to be a collab. Please ask.

**Comment by harshithpabbati** [comment]
Yeah I will do it. But takes time for sure. I have exams now. I will be active in reviewing pr's and to have discussion's .

**Comment by HarshKhandeparkar** [comment]
sure. TY. I will also be busy for a few days

**Comment by harshithpabbati** [comment]
Will use sass. Instead of scss. Because the syntax constraints are very less and can be written simply.

**Comment by HarshKhandeparkar** [comment]
I didn't get you. Do you want to change the file extension?

**Comment by harshithpabbati** [comment]
Yeah

**Comment by HarshKhandeparkar** [comment]
You can open a PR if you want. You can do that in the browser too.

**Comment by HarshKhandeparkar** [comment]
BTW, I am working on the module selector.

**Comment by HarshKhandeparkar** [comment]
@vaibhavmatta you can report your status her so that everyone knows. Thanks!
